sharepoint/sharepoint-ps/Microsoft.Online.SharePoint.PowerShell/Add-SPOContainerUser

```diff
@@ -0,0 +1,139 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/add-spocontaineruser
+applicable: SharePoint Online
+title: Add-SPOContainerUser
+schema: 2.0.0
+author: ShreyasSar26
+ms.author: shsaravanan
+ms.reviewer:
+---
+
+# Add-SPOContainerUser
+
+## SYNOPSIS
+
+Adds a user to a SharePoint Embedded container with a specified role.
+
+## SYNTAX
+
+```
+Add-SPOContainerUser [-ContainerId] <String> -LoginName <String> -Role <String> [<CommonParameters>]
+```
+
+##DESCRIPTION
+
+Assigns a user to a defined role within a SharePoint Embedded container. You must specify the container ID, the user's login name, and the role to be assigned. This cmdlet is used to grant access and permissions based on role membership.
```

Comment:
>You must specify the container ID, the user's login name, and the role to be assigned.  [](http://example.com/codeflow?start=73&length=87)

this sentence is redundant -- the SYNTAX part already clarifies these params are mandatory